target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8074-wxr-5950ax12.dts

```diff
@@ -0,0 +1,366 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "ipq8074.dtsi"
+#include "ipq8074-hk-cpu.dtsi"
+#include "ipq8074-ess.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Buffalo WXR-5950AX12";
+	compatible = "buffalo,wxr-5950ax12", "qcom,ipq8074";
+
+	aliases {
+		serial0 = &blsp1_uart5;
+		led-boot = &led_power_white;
+		led-failsafe = &led_power_red;
+		led-running = &led_power_white;
+		led-upgrade = &led_power_white;
+		label-mac-device = &dp5_syn;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs-append = " ubi.mtd=user_property root=/dev/ubiblock1_0";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led-0 {
```

Comment:
`led-N` isnt really descriptive, since the functions are known just make then `led-function`

Reply:
It's based on DT binding documentation in Linux Kernel ([leds-gpio.yaml](https://elixir.bootlin.com/linux/v5.15.92/source/Documentation/devicetree/bindings/leds/leds-gpio.yaml)).

Reply:
Ok, thanks